stable/mssql-linux/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: SQL Server 2017 Linux Helm Chart
 name: mssql-linux
-version: 0.1.8
+version: 0.1.9
```

Comment:
You're adding a new feature (make use of an existing claim, nodeselector), so please bump the minor version of the Chart.

Reply:
Good point.  Completed.